src/PlanetData.js

```diff
@@ -43,18 +48,21 @@ PlanetData.prototype = {
         red = c.r * d.x;
         green = c.g * d.x;
         blue = c.b * d.x;
-        this.mapData.pixels[y * this.mapData.width + x] = Phaser.Color.packPixel(red, green, blue, alpha);
+        i = 4 * (y * this.width + x);
+        this.mapData.data[i] = red;
+        this.mapData.data[i + 1] = green;
+        this.mapData.data[i + 2] = blue;
+        this.mapData.data[i + 3] = alpha;

```

Comment:
I'm not a huge fan of keeping the `data` in a 1d array and using a math equation to figure out the index `y * this.width + x`. Can we make it a 2d array?